docs/content/reference/resource-schema/core-schema/environment-schema/index.md

```diff
@@ -44,6 +46,32 @@ Details on what to run and how to run it are defined in the `container` property
 | kind | y | The kind of identity. 'azure.com.workload' is currently only supported. | `'azure.com.workload'` |
 | oidcIssuer | n | The [OIDC issuer URL](https://azure.github.io/azure-workload-identity/docs/installation/self-managed-clusters/oidc-issuer.html) for your Kubernetes cluster. | `'{IssuerURL}/.well-known/openid-configuration'` |
 
+### recipeConfig
+
+| Key  | Required | Description | Example |
+|------|:--------:|-------------|---------|
+| terraform | y | Configuration for Terraform Recipes. Controls how Terraform plans and applies templates as part of Recipe deployment. | [See below](#terraform-properties)
+| env | n | Environment variables injected during Terraform Recipe execution for the recipes in the environment. | [TODO](#TODO)
```

Comment:
Need help with what an example would be here

Reply:
Draft:
### env Properties

| Key  | Required | Description | Example |
|------|:--------:|-------------|---------|
| \<user-defined key-value pairs\> | n | User-defined environment variables. `port: '2222'`